src/core/python/utility/pyboost_cv_converter.cpp

```diff
@@ -435,7 +285,60 @@ void matFromNDArrayBoostConverter::construct(
     data->convertible = storage;
 }
 
+// pt converters
+
+PyObject* pointToTupleBoostConverter::convert(cv::Point2f const& pt)
+{
+    PyObject* elem1 = PyFloat_FromDouble(pt.x);
+    PyObject* elem2 = PyFloat_FromDouble(pt.y);
+    PyObject* tuple = PyTuple_Pack(2, elem1, elem2);
+    return tuple;
+}
+
+pointFromTupleBoostConverter::pointFromTupleBoostConverter()
+{
+    boost::python::converter::registry::push_back(convertible, construct,
+                                                  boost::python::type_id<cv::Point2f>());
+}
+
+// check if PyObject is a tuple of numbers and can be converted to point
+void* pointFromTupleBoostConverter::convertible(PyObject* object)
+{
+    // check if we have a tuple of size 2
+    if (!PyTuple_Check(object) || PyTuple_Size(object) != 2) return NULL;
+
+    // check if the elements are numbers
+    PyObject* elem1 = PyTuple_GetItem(object, 0);
+    PyObject* elem2 = PyTuple_GetItem(object, 1);
+    if (!PyNumber_Check(elem1) || !PyNumber_Check(elem2)) return NULL;
+
+    return object;
+}
+
+// construct point2f from a tuple
+void pointFromTupleBoostConverter::construct(
+  PyObject* object, boost::python::converter::rvalue_from_python_stage1_data* data)
+{
+    PyObject* elem1 = PyTuple_GetItem(object, 0);
+    PyObject* elem2 = PyTuple_GetItem(object, 1);
+
+    float elem1f = (float)PyFloat_AsDouble(elem1);
+    float elem2f = (float)PyFloat_AsDouble(elem2);
+
+    // Grab pointer to memory into which to construct the new QString
+    void* storage = (
+      (boost::python::converter::rvalue_from_python_storage<cv::Point2f>*)
+      data)->storage.bytes;
+
+    // in-place construct the new QString using the character data
```

Comment:
ditto. Just remove the inline comments in this function whatsoever please, they don't much of a new information anyway.